lib/templates/plugin.server.js

```diff
@@ -4,6 +4,18 @@ import colorSchemeComponent from './color-scheme'
 Vue.component('<%= options.componentName %>', colorSchemeComponent)
 
 export default function (ctx, inject) {
+  const script = `<%= options.script %>`
```

Comment:
Perf: script var and script object can be outside of function to avoid recreation on ssr calls